src/components/Application/ApplicationStart.tsx

```diff
@@ -18,7 +19,8 @@ const ApplicationStart: React.FC<ApplicationStartProps> = (props) => {
   return template ? (
     <Segment.Group style={{ backgroundColor: 'Gainsboro', display: 'flex' }}>
       <Button
-        as="href"
+        as={Link}
+        to={`/applications`}
```

Comment:
Shouldn't this link to the specific application type?
`/applications?type={applicationType}` Something like that?

I know we haven't implemented list query routing yet, but it can't hurt to send it to what will be the right place.

Reply:
Yes. I guess that's better, I will change to this now

Reply:
Done: [ebc4a9f](https://github.com/openmsupply/application-manager-web-app/pull/234/commits/ebc4a9f06ba22c76c3f3706f83266848219ae7a6)